x-pack/plugins/cases/common/api/metrics/case.ts

```diff
@@ -76,15 +96,15 @@ export const SingleCaseMetricsRequestRt = rt.strict({
   /**
    * The metrics to retrieve.
    */
-  features: rt.array(rt.string),
+  features: rt.array(CaseMetricsFeatureFieldRt),
```

Comment:
If I remember correctly the `mttr` feature is not available for the single case request.

Reply:
Thanks for this catch, I differentiate now between Single Case or Cases Metrics Features. 👍